cmd/query/app/apiv3/http_gateway.go

```diff
@@ -136,7 +136,10 @@ func (h *HTTPGateway) getTrace(w http.ResponseWriter, r *http.Request) {
 	if h.tryParamError(w, err, paramTraceID) {
 		return
 	}
-	trc, err := h.QueryService.GetTrace(r.Context(), traceID)
+	query := spanstore.GetTraceParameters{
+		TraceID: traceID,
```

Comment:
copy timestamps from request

Reply:
Added TODO, since API changes will be in another PR